AEPCore/Sources/configuration/Configuration.swift

```diff
@@ -146,8 +147,12 @@ class Configuration: NSObject, Extension {
                 // If downloading config failed try again later
                 guard let self = self else { return }
                 Log.trace(label: self.name, "Downloading config failed, trying again")
-                self.retryQueue.asyncAfter(deadline: .now() + 5) {
-                    self.processConfigureWith(appId: appId, event: event, sharedStateResolver: sharedStateResolver)
+                self.retryQueue.asyncAfter(deadline: .now() + (self.retryConfigurationCounter * 2)) {
```

Comment:
Can we change it to `self.retryConfigurationCounter * 5` to increase the time between initial retries.